README.md

```diff
@@ -98,12 +91,61 @@ Install the unarchived tar.gz file
 ansible-galaxy collection install cisco-aci-1.0.0.tar.gz —-force
 ```
 
-### See Also
+## Use Cases
+
+Once the collection is installed, you can use it in a playbook by specifying the full namespace path to the module, plugin and/or role.
+
+### Adding a new EPG
+
+```yml
+- hosts: aci
+  gather_facts: no
+
+  tasks:
+  - name: Add a new EPG
+    cisco.aci.aci_epg:
+      hostname: apic
+      username: admin
+      password: SomeSecretPassword
+      tenant: production
+      ap: intranet
+      epg: web_epg
+      description: Web Intranet EPG
+      bd: prod_bd
+    delegate_to: localhost
+```
+
+## Optimizing Playbooks
+
+To find out more about optimizing playbook execution, please refer to the [Optimizing Playbooks](docs/optimizing.md) documentation.
+
+## Testing
 
-- [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
+Integration tests for each module in the cisco.aci collections are executed against the following ACI versions:
 
-## Contributing to this collection
+- 4.2(7s)
+- 5.2(7g)
+- 6.0(2h)
+
+## Contributing
 
 Ongoing development efforts and contributions to this collection are tracked as issues in this repository.
 
 We welcome community contributions to this collection. If you find problems, need an enhancement or need a new module, please open an issue or create a PR against the [Cisco ACI collection repository](https://github.com/CiscoDevNet/ansible-aci/issues).
+
+## Support
+
+This collection has supports the ACI versions listed in the Testing section.
```

Comment:
Hmm LDOS for 3.2 is 30th Nov 2024. Did we remove testing this version too early? We can't really say we support any ACI version that is within LDOS. However, I'll change it anyway because its only in a month or so.

Reply:
Re-worded this section and removed the explicit mention of 4.2+ support.